### PR TITLE
BZ1301453 - Form is not fully shown for User Task in Task list.

### PR DESCRIPTION
**Pull request by nmirasch**
Put the form buttons in a fixed location leaving the form in a ScrollPanel.

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@nmirasch unfortunately a fixed height isn't ideal. We would have to find a way to calculate it dynamically.

**Comment by nmirasch** [comment]
@cristianonicolai updated!

File: jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/details/multi/ProcessInstanceDetailsMultiPresenter.java [hunk 1]
```diff
@@ -248,17 +248,17 @@ public void callback(List<NodeInstanceSummary> completedNodes) {
                                                             .addParameter("completedNodes", completedNodeParam.toString())
                                                             .addParameter("readOnly", "true")
                                                             .addParameter("processId", processId)
-                                                            .addParameter("deploymentId", selectedDeploymentId)));
+                                                            .addParameter("processInstanceId", selectedProcessInstanceId)));
```
Comment (cristianonicolai) [inline jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/details/multi/ProcessInstanceDetailsMultiPresenter.java]: Seems like this is breaking the process instance diagram view. I think these changes should be done but perhaps in a separated PR as they are somehow unrelated to the main issue.

**Comment by nmirasch** [comment]
@cristianonicolai, updated

**Comment by cristianonicolai** [comment]
+1 thanks @nmirasch!

**Pull request merged**